packages/amplify_datastore/pigeons/native_plugin.dart

```diff
@@ -87,3 +107,23 @@ class LegacyCredentialStoreData {
   String? refreshToken;
   String? idToken;
 }
+
+class NativeGraphQLResponse {
+  String? payloadJson;
+  String? errorsJson;
+}
+
+class NativeGraphQLSubscriptionResponse {
```

Comment:
Are all of the params actually nullable? Same for `NativeGraphQLRequest`

Reply:
Some are, I can update them.